front/packages/ui/src/browse/grid.tsx

```diff
@@ -157,7 +157,10 @@ export const ItemGrid = ({
 			</PosterBackground>
 			<Skeleton>
 				{isLoading || (
-					<P numberOfLines={1} {...css([{ marginY: 0, textAlign: "center" }, "title"])}>
+					<P
+						numberOfLines={subtitle ? 1 : 2}
```

Comment:
If I recall correctly, this is only used for collections. Is this intended?

PS: I was thinking of adding something like `3 items` as a subtitle for collection, so that would cancel that.

Reply:
For now, yes, this would be mainly applicable to collections.

However, would it be possible for a show not to have a year? This would be another 'use case'.
Let me know what you think

Reply:
True, it is possible, but I have never seen it happens. If this was intended, that's good.